int8 device_number
---
FestoErrors resp
